### PR TITLE
format backupfile

### LINKED ISSUE #54: Account service: generate .bin file
Need a method in `Account` service that generates .bin file via brainkey

### PR DESCRIPTION
**Pull request by ololokiras**
create backup .bin file 
closes #54

### CODE REVIEW

File: src/modules/acc/actions.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-// temp
-import { Aes } from 'bitsharesjs';
+import { Aes, PublicKey, key } from 'bitsharesjs';
```
Comment (roma219) [inline src/modules/acc/actions.js]: not used?

File: src/services/api/account.js [hunk 2]
```diff
@@ -188,11 +189,86 @@ export const createAccountBrainkey = async ({ name, brainkey, email }) => {
...
+      backup_date: new Date().toString(),
+      brainkey_backup_date: new Date(),
+      brainkey_pubkey: brainkeyPubkey,
+      brainkey_sequence: 0,
+      chain_id: config.MAIN_NET_CHAINID,
+      created: new Date(),
+      encrypted_brainkey: encryptedBrainkey,
+      encryption_key: encryptionKey,
+      id: 'default',
+      last_modified: new Date(),
```
Comment (roma219) [inline src/services/api/account.js]: No need to call `new Date()` 3 times

File: src/services/api/account.js [hunk 3]
```diff
@@ -188,11 +189,86 @@ export const createAccountBrainkey = async ({ name, brainkey, email }) => {
...
+      ]);
+      resolve(backup);
+    });
+  });
+}
+
+// comments get from bitshares-ui
+export const generateBackupFileFromBrainkey = async ({ brainkey, password }) => {
+  const accountId = await getAccountIdByBrainkey(brainkey);
+  const { name } = await getUser(accountId);
```
Comment (roma219) [inline src/services/api/account.js]: why do you need to fetch `accountId` and `name`, when it is already in store?

File: src/services/api/account.js [hunk 4]
```diff
@@ -188,11 +189,86 @@ export const createAccountBrainkey = async ({ name, brainkey, email }) => {
...
+  const passwordPrivate = PrivateKey.fromSeed(password);
```
Comment (roma219) [inline src/services/api/account.js:222]: Why do you need a password? Assume the wallet is unlocked

**Review by roma219 — CHANGES_REQUESTED** [review]

File: src/services/api/account.js (hunk 4 above)
Comment (ololokiras) [reply src/services/api/account.js:222]: I need password as field for encrypt in backup file for bitshares-ui users

**Review by roma219 — APPROVED** [review]

**Pull request merged**